product_docs/docs/pgd/5/deployments.mdx

```diff
@@ -14,15 +14,15 @@ You can deploy and install EDB Postgres Distributed products using the following
 
 |                                            | <center>TPA</center> |         <center>BigAnimal</center>         | <center>Kubernetes<br/>(preview)</center> |
 |--------------------------------------------|:------------------------:|:------------------------------------------:|:-----------------------------------------:|
-| Single region                              |            ✅             |                     ✅                      |                     ✅                     |
+| Single region                              |            ✓             |                     ✓                      |                     ✓                     |
```

Comment:
@djw-m think the green checkmarks make it easier to scan at a glance - with the checkmarks the same color as the dashes, it makes it harder to understand what is/isn't available yet